tests/execution/test-set-cookie-headers.md

```diff
@@ -0,0 +1,42 @@
+# Set Cookie Header
+
+```graphql @server
+schema
+  @server(port: 8080, graphiql: true, hostname: "0.0.0.0", headers: {setCookies: true})
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
+  query: Query
+}
+
+type Query {
+  user(id: Int!): User @http(path: "/users/{{args.id}}")
+}
+type User {
+  id: Int!
+  name: String!
+  username: String!
+  email: String!
+  phone: String
+  website: String
+}
+```
+
+```yaml @mock
+- request:
+    method: GET
+    url: http://jsonplaceholder.typicode.com/users/1
+
+  response:
+    status: 200
+    body:
```

Comment:
response should have a `set-cookie` header.

Reply:
Why? It’s response from jsonplaceholder I dont think we have code that does this

Reply:
Was that the original issue?

Reply:
`set-cookie` is a response header and not a request header. It is received from the upstream API responses. The client will never send a `set-cookie` header.